src/main/java/ru/vsu/csd/datatransferservice/domain/Movie.java

```diff
@@ -0,0 +1,152 @@
+package ru.vsu.csd.datatransferservice.domain;
+
+import jakarta.persistence.*;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Random;
+
```

Comment:
You can use Lombok annotations for constructor and setters/getters here if you want